rubric.md

```diff
@@ -0,0 +1,69 @@
+# So you want the IWG to run a service…
+
+⚠️  This document is maintained by the IWG and is likely to change as we learn
+from the process of actually carrying out our mandate.
+
```

Comment:
Would a bridge paragraph here, briefly restating the mandate and explaining that both the meaning of "run a service" and list of what we _don't_ do derive directly from that?

Reply:
right - does "run" include "maintain"? Or just "host"? etc.

Reply:
I tweaked this a bit.